tests.py

```diff
@@ -1815,3 +1815,21 @@ def test_missing_source_files(self):
                 "some files listed as being under source control are missing:\n"
                 "  missing.py",
                 sys.stderr.getvalue())
+
+    def test_broken_symbolic_link(self):
+        from check_manifest import check_manifest
+        self._create_repo_with_code()
+        os.symlink(os.path.join(self.tmpdir, "some-file"),
+                   os.path.join(self.tmpdir, "some-symbolic-link"))
+        self._add_to_vcs('some-file')
+        self._add_to_vcs('some-symbolic-link')
+        self._add_to_vcs('MANIFEST.in')
+        with open(os.path.join(self.tmpdir, "MANIFEST.in"), "w") as f:
+            f.write("include some-symbolic-link\ninclude some-file\n")
+        self.assertTrue(check_manifest())
+        self._vcs._commit()
+        os.unlink('some-file')
+        check_manifest()
+        self.assertIn("some symbolic links listed as being under source control are pointing to missing files:\n"
+                      f"  some-symbolic-link -> {os.path.join(self.tmpdir, 'some-file')} (missing)",
```

Comment:
... because then you might be able to use 

```suggestion
                      "  some-symbolic-link -> some-file (missing)",
```

here.

Of course, Windows being Windows, there's a chance that even relative symlinks will be converted into those weird absolute UNC paths, in which case plan B is 

```python
# On Windows os.readlink() returns weird UNC paths so we cannot hardcode os.path.join(self.tmpdir, "...") here
link_target = os.readlink(os.path.join(self.tmpdir, "some-file"))
```

followed by using 

```suggestion
                      f"  some-symbolic-link -> {link_target} (missing)",
```

here.